app/controllers/addresses_controller.rb

```diff
@@ -2,7 +2,7 @@ class AddressesController < ApplicationController
   before_action :customer?, only: %i[index edit update destroy create new]
   before_action :set_address, only: %i[edit update destroy]
   def index
-    @addresses = current_user.address
+    @addresses = current_user.addresses
```

Comment:
This doesn´t need to change but as a recommendation if this is a singular thing why use a plural variable. It may lead to confusion when using it

Reply:
if we leave it in singular, because of the relation they have in the models, it gives us error.

Reply:
I mean the @ variable